### DISCUSSION TITLE
Did commands change with version 1.3?

### DISCUSSION THREAD

**Discussion by Lalomonx** [keefo/NeewerLite#41] [Q&A]
I had set my color through commands but they are not changing color anymore, just brightness, I'm not great with code but I thing RGB value should be changed to HUE?

**Comment by keefo** [comment]
Due to recent refactoring, the cmd system is broken. Will bring it back asap. And the music following feature as well.

**Comment by keefo** [comment]
version 1.4 is released. The command should work on it. It also bring minor change to some commands. Please read the home page doc to see the change.
